zipline/pipeline/loaders/utils.py

```diff
@@ -307,10 +300,10 @@ def get_values_for_date_ranges(zip_date_index_with_vals,
         returns a pd.Series with the values indexed by the dates.
     vals_for_date_intervals : list
         A list of values for each date interval in `date_intervals`.
-    date_intervals : list
-        A list of pairs of dates, where each pair represents a date interval
-        that corresponds to the value at the same index in
-        `vals_for_date_intervals`.
+    starts : list
```

Comment:
Can we just pass a DatetimeIndex instead of parsing all the strings individually here?